### PR TITLE
[docs] Update get_table_* reference in Chain API swagger file

### PR DESCRIPTION
**Pull request by deckb**
## Change Description

Updated get_table_* documentation swagger file.

## Change Type
**Select ONE**
- [x] Documentation

- [ ] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Comment by deckb** [comment]
I have not called `thorka` to check the output yet. Looking to do that soon.

**Review by iamveritas — CHANGES_REQUESTED** [review]
changes look good to me, but please find a way to run thorka locally and validate how these changes get parsed and generated.
one of dev rel team members can help or if you want to install and run thorka yourself please do it.

File: plugins/chain_api_plugin/chain.swagger.yaml [hunk 1]
```diff
@@ -488,10 +494,25 @@ paths:
...
-                upper_bound:
-                  type: string
-                lower_bound:
-                  type: string
+                  lower_bound:
+                    type: string
+                    description: Filters results to return the first element that is not less than provided value in set
+                  upper_bound:
+                    type: string
+                    description: Filters results to return the first element that is greater than provided value in set
```
Comment (lparisc) [inline plugins/chain_api_plugin/chain.swagger.yaml:502]: These two properties should be intended two spaces left

**Review by lparisc — CHANGES_REQUESTED** [review]

File: plugins/chain_api_plugin/chain.swagger.yaml (hunk 1 above)
Comment (deckb) [reply plugins/chain_api_plugin/chain.swagger.yaml]: Ended up making the changed and pushing

**Review by lparisc — APPROVED** [review]

**Review by iamveritas — APPROVED** [review]

**Comment by lparisc** [comment]
Replaced by PR #9627

**Pull request closed without merging**